core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -932,11 +956,9 @@ trait EntityComponent {
     // copy entities from one workspace to another, checking for conflicts first
 
     def checkAndCopyEntities(sourceWorkspaceContext: Workspace, destWorkspaceContext: Workspace, entityType: String, entityNames: Seq[String], linkExistingEntities: Boolean, parentSpan: Span = null): ReadWriteAction[EntityCopyResponse] = {
+
       def getHardConflicts(workspaceId: UUID, entityRefs: Seq[AttributeEntityReference]) = {
-        val batchActions = createBatches(entityRefs.toSet).map(batch => getEntityRecords(workspaceId, batch))
-        DBIO.sequence(batchActions).map(_.flatten.toSeq).map { recs =>
-          recs.map(_.toReference)
-        }
+       getEntityRecords(workspaceId, entityRefs.toSet)
       }
```

Comment:
looking at this even closer, the only place that `getHardConflicts` is called only uses the entity type and entity name from the retrieved records. So, `getEntityRecords` is overkill; it returns columns we don't need. `getHardConflicts` could call `activeActionForRefs` instead, which only returns the type&name columns. What's your take on this? Is it worth changing?